classScreen.js

```diff
@@ -0,0 +1,34 @@
+(function(global) {
+    "use strict";
+
+    function Screen(element) {
+        this.element = element;
+        this.reset = false;
+    }
+
+    Screen.prototype.resetOnNextInput = function() {
+        this.reset = true;
+    }
+
+    Screen.prototype.setNumber = function(number) {
+        this.element.value = number;
+    }
+
+    Screen.prototype.getNumber = function() {
+        return this.element.value;
+
+    };
+
+    Screen.prototype.addDigit = function(digit) {
+
+        if (this.reset === true) {
+            this.element.value = digit;
+            this.reset = false;
+        } else {
+            this.element.value += digit;
+        }
+    }
+    
+    global.Screen = Screen;
```

Comment:
Lets not pollute the global -- window scope anymore, since now we have support for namespaces which are solving this problem. Use provide here to export the class, and require afterwards when you need it:
```
provide('calculator.output.Screen', Screen);
```